box.json

```diff
@@ -0,0 +1,20 @@
+{
+    "force-autodiscovery": true,
+    "directories": [
+        "config"
+    ],
+    "files": [
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/AbstractFixerTestCase.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/AbstractIntegrationCaseFactory.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/AbstractIntegrationTestCase.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/Assert/AssertTokensTrait.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/IntegrationCase.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/IntegrationCaseFactory.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/IntegrationCaseFactoryInterface.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/Test/InternalIntegrationCaseFactory.php",
+        "vendor/friendsofphp/php-cs-fixer/tests/TestCase.php"
```

Comment:
can't it be simplified into:

```json
"directories: [
  "config",
  "vendor/friendsofphp/php-cs-fixer/tests/Test"
}
```

?

Reply:
It looks like that would work 👍

Reply:
except  "vendor/friendsofphp/php-cs-fixer/tests/TestCase.php"

Reply:
Ah. 🤔

Reply:
Does it matter though?

Reply:
Probably as the testcases in `tests/Test` extend it.